### PR TITLE
Fix `Solid` nickname color

### PR DESCRIPTION
**Pull request by neilalexander**
This was accidentally changed to the action color in #232.

### CODE REVIEW

**Comment by casperstorm** [comment]
Hmm, this was because the nicknames in chat was `action` if solid was selected, but nicknames in sidemenu was `text`. I think we should consider `accent` for both instead:

Screenshot 2024-03-17 at 06 42 17

**Comment by neilalexander** [comment]
This is probably one of those subjective/theme-specific things — to my eyes, the "blending together" was what made the nick list disappear into the periphery rather than stand out too much. For example, this stands out quite a lot:

Screenshot 2024-03-17 at 10 53 39

... whereas this felt more in-keeping to me with the rest of the theme:

Screenshot 2024-03-17 at 10 54 37

Accent colour might be OK, it doesn't look as though it's used by much else apart from text selection?

Another alternative is that we put it to `text` and add some permanent alpha to it, so that it's subtly different but not by much, for example at alpha 0.6:

Screenshot 2024-03-17 at 10 57 21

**Comment by casperstorm** [comment]
Colors are hard :)
Perhaps the solution is simply to do `text.base` for both nicklist and nicknames in buffer.
It will blend the most, which i assume people who use this option want.

**Comment by neilalexander** [comment]
Right now there are separate `colors:` options for the buffer and nick list, so they can be controlled separately. That seems like a nice option to keep? Then you can have colours in the buffer but not nick list (like I do), both or neither.

Presumably both use the `nickname()` function?

**Comment by casperstorm** [comment]
> Right now there are separate `colors:` options for the buffer and nick list, so they can be controlled separately. That seems like a nice option to keep? Then you can have colours in the buffer but not nick list (like I do), both or neither.
> 
> Presumably both use the `nickname()` function?

Correct, but initially the nickname color in the buffer was `action` and the colors in nicklist was `text`.
Then in `main` both is `action`, and this PR changes both to `text`. 😄 

I am good with that, now that I've given it some thoughts.

File: src/theme/text.rs [hunk 1]
```diff
@@ -49,14 +50,23 @@ pub fn transparent(theme: &Theme) -> Appearance {
...
 pub fn nickname(theme: &Theme, seed: Option<String>, transparent: bool) -> Appearance {
+    let dark_theme = theme.colors().is_dark_theme();
+
+    if seed.is_none() {
+        let mut color = theme.colors().text.base;
+        if transparent {
+            color = alpha(color, if dark_theme { 0.2 } else { 0.4 })
+        }
+        return Appearance { color: Some(color) };
+    }
```
Comment (casperstorm) [inline src/theme/text.rs:61]: What about:

```rust
    if seed.is_none() {
        let color = match transparent {
            true => theme.colors().text.med_alpha,
            false => theme.colors().text.base,
        };

        return Appearance { color: Some(color) };
    }
```

Comment (neilalexander) [reply src/theme/text.rs:61]: That's much nicer and more compact, will do that!

**Comment by casperstorm** [comment]
Could you also add a small line to CHANGELOG? Then it should be good to go.

**Comment by neilalexander** [comment]
Done!

**Review by casperstorm — APPROVED** [review]
Thanks a bunch @neilalexander

**Pull request merged**